js/cljdoc.js

```diff
@@ -126,6 +126,18 @@ function restoreSidebarScrollPos() {
   localStorage.removeItem("sidebarScrollPos");
 }
 
+if (document.querySelector(".main-scroll-view")) {
+  document.getElementById("meta-icon").onclick = function() {
```

Comment:
Let's prefix JS-relevant classes with `js--` and avoid too generic names like `close` to make it easier to relate things and avoid conflicts. 

Maybe some thing like:

- `#js--meta-close`
- `#js--meta-dialog`
- `#js--meta-icon`

Also consider exporting this chunk as a function and calling it in `index.js`. Most of the existing JS files just define functions which are then called in `index.js` — this way it's easier to trace the code and see what is happening at a glance.